RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -166,48 +166,37 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     float y_center = circleCenterY / 100;
     float pt = 2 * k2Rinv1GeVf * circleRadius;  //k2Rinv1GeVf is already in cm^(-1)
 
-    //determine the charge
-    int charge = 0;
-    if ((x2 - x1) * (y3 - y1) - (y2 - y1) * (x3 - x1) > 0)
-      charge = -1;
-    else
-      charge = 1;
-
+    float cross = (x2 - x1) * (y3 - y1) - (y2 - y1) * (x3 - x1);
+    int charge = -1 * ((int)copysignf(1.0f, cross));
+      
     //get the absolute value of px and py at the initial point
     float px = 2 * k2Rinv1GeVf * alpaka::math::abs(acc, (y_init - y_center)) * 100;
     float py = 2 * k2Rinv1GeVf * alpaka::math::abs(acc, (x_init - x_center)) * 100;
```

Comment:
doesn't this work all the way to `//But if the initial T3 curve` :
```suggestion
    //get the absolute value of px and py at the initial point
    float px = 2 * charge * k2Rinv1GeVf * (y_init - y_center) * 100;
    float py = - 2 * charge * k2Rinv1GeVf * (x_init - x_center) * 100;
```

Reply:
did this suggestion not work?
I don't understand why we need to compute with an abs and then undo it on the next line

Reply:
Yes, the suggested change works. I have changed in a new commit and verified the physics performance. Thanks